srcpkgs/cantata/template

```diff
@@ -1,20 +1,26 @@
 # Template file for 'cantata'
 pkgname=cantata
-version=2.5.0
-revision=2
+version=3.2.0
+revision=1
 build_style=cmake
-hostmakedepends="pkg-config qt5-qmake qt5-host-tools qt5-tools"
-makedepends="qt5-devel qt5-tools-devel qt5-svg-devel qt5-xmlpatterns-devel
- qt5-plugin-mysql qt5-plugin-pgsql qt5-plugin-sqlite qt5-plugin-odbc
- qt5-plugin-tds vlc-devel zlib-devel ffmpeg6-devel taglib-devel
- taglib-extras-devel libmtp-devel mpg123-devel libcdparanoia-devel libcddb-devel
- speex-devel eudev-libudev-devel udisks2-devel libmusicbrainz5-devel
- qt5-multimedia-devel libcdio-paranoia-devel"
-depends="media-player-info qt5-plugin-sqlite"
+configure_args="-DENABLE_HTTP_STREAM_PLAYBACK=ON
+ -DENABLE_LIBVLC=OFF
+ -DENABLE_SIMPLE_MPD_SUPPORT=ON
+ -DCMAKE_BUILD_TYPE=None
+ -DCMAKE_INSTALL_LIBEXECDIR=/usr/bin"
```

Comment:
cantata-replaygain and cantata-tags don't have to be installed alongside the main executable. I believe setting this parameter this way will lead to a folder being created under /usr/bin which is probably not wanted.

I'm not familiar with Void's semantics on the use of libexec, but CMake suggests that LIBEXECDIR be relative (it is then relative to CMAKE_INSTALL_PREFIX.) It is probably best to leave it as default (`libexec`) or if needed, `lib64/libexec` (wherever the libexec dir is relative to /usr, the CMAKE_INSTALL_PREFIX.) Then these features will work properly.